resources/helm/v2.4/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1810,7 +1809,6 @@ spec:
                 operator: In
                 values:
                 - "amd64"
-                - "arm64"
```

Comment:
Question: why we are deleting the arm64 from here? This will not break the support for the istiod to deploy on that arch?

Reply:
@fjglira I will add it back in. It was automatically deleted with `make gen`